### ISSUE TITLE
Address "Credential" vs "VerifiableCredential"

### ISSUE THREAD

**Issue by OR13** [w3c/vc-data-model#1126]
https://w3c.github.io/vc-data-model/#types

<img width="844" alt="Screen Shot 2023-05-09 at 6 31 46 PM" src="https://github.com/w3c/vc-data-model/assets/8295856/23f87cdc-b8cb-45ca-9f2a-f6ead787786a">

Table indicates these are separate `types`, but the working group has since resolved not to define them separately, and every property that is valid for `credential` is also valid for `verifiable credential`.

**Comment by OR13** [comment]
See https://github.com/w3c/vc-data-model/pull/1057

**Comment by awoie** [comment]
This means, we should remove "Credential" entirely from the VCDM? IMO, it is confusing to have both in the VCDM, especially since the securing mechanisms are no longer part of the VCDM.

**Comment by OR13** [comment]
@awoie see https://github.com/w3c/vc-data-model/pull/1129

Data Integrity proofs are in the v2 context, and in all the examples, I think your assessment is not accurate... the current spec has defects in both directions of the argument.

**Comment by awoie** [comment]
@OR13 I fully agree we should fix that.

**Comment by David-Chadwick** [comment]
There is clearly a difference between an object with a proof, meaning that it is verifiable, and an object without a proof, meaning that it is not verifiable. Since we have both of these objects in our specifications then we should be clear which term we apply to each of these objects. We cannot use the same term to apply to two objects because this introduces ambiguity.
I thought we already had an agreement that the object without the proof was to be termed "credential" and the object with the proof was to be termed "verifiableCredential"?

**Comment by OR13** [comment]
I agree with ted.

I think we are making a mistake in the current spec, referring to credential and verifiable credentials separately... 


Both can have proof and neither might verify, they are not useful terms for distinguishing attributes of the data model, since they support the exact same properties.

We should avoid "credential" and focus on "claims" and "verifiable credential", or we should drop the word "verifiable" from everything but the spec title, and just have claims and credentials...

I was rereading WebAuthN yesterday, and I think we have missed a huge opportunity to align better with it, due to our terminology coming from RDF instead of security vocabulary.

The minimal VerifiablePresentation is just a proof from an authenticator (signature from a credential). 

The difference is web authn credentials are bound to a single RP (2 party model) and VCDM is not like that, and has more serious privacy issues because of this.

Our concept of "holder binding / confidence method" is their concept of allowedCredentials but scoped again to only authentication.

If WebAuthN didn't need the word verifiable to be successful, maybe we don't either.

**Comment by andresuribe87** [comment]
> we should drop the word "verifiable" from everything but the spec title

What are the implications of dropping it from the spec title as well?

If the working group has agreed that anything can be verifiable, then there isn't much value in using that as a qualifier.

**Comment by David-Chadwick** [comment]
@OR13 said `Both can have proof `. This is clearly where there is a semantic disconnect in the group which needs to be resolved. One view is that a credential is the VCDM data object without a proof (external or internal) and that a verifiable credential is a credential with a proof. @OR13 's view appears to be that both objects are identical. If the latter is true we do not need two terms as this is confusing. If the former is true, then we do. 
To the chairs. Can we resolve this with a vote please?
Note. Whether the proof is verifiable or not is irrelevant to the discussion of what data object(s) the terms refer to.

**Comment by OR13** [comment]
@David-Chadwick we voted on this already... 

It is the RDF data model that makes the word credential meaningless and indistinguishable from "VerifiableCredential".

I didn't agree with the decision, or the data modeling design choices, but I don't think it should be reopened.

The section of the spec that defines proof is wrong, and needs to be updated. There is no meaningful difference between credential and verifiable credential... It will continue to confuse people.

I suggest @msporny or @dlongley open the PR to fix this, since they seem to understand the data model and both objected to the working group defining credential in RDF in way that would distinguish it from VerifiableCredential.

**Comment by OR13** [comment]
I also look forward to seeing this addressed

**Comment by brentzundel** [comment]
Is this a duplicate of #1009?

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-07-12-vcwg#section5-5) on 2023-07-12

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 5.5. Address "Credential" vs "VerifiableCredential" (issue vc-data-model#1126)

_See github issue [vc-data-model#1126](https://github.com/w3c/vc-data-model/issues/1126)._




**Brent Zundel:** address credential vs verifiable credential.  
… we tried to address this, not sure what is left.  

**Manu Sporny:** this is before CR.  
… I can be the backstop on it.  
… we have... 702 statements of the word credential "446" for verifiable credential.  
… editors will need to address "credential vs verifiable credential".  
… there are some issues remaining...  
… we need to adjust the language of the spec, but its a lot of work.  

**Brent Zundel:** thanks, I will try to help.  

</details>

**Comment by msporny** [comment]
I have opened PR #1211 in an attempt to address this issue. Once #1211 is merged, we will close this issue.

**Comment by msporny** [comment]
PR #1211 has been merged, closing.
